notify/grafana_alertmanager.go

```diff
@@ -191,16 +197,17 @@ func (c *GrafanaAlertmanagerConfig) Validate() error {
 func NewGrafanaAlertmanager(tenantKey string, tenantID int64, config *GrafanaAlertmanagerConfig, peer ClusterPeer, logger log.Logger, m *GrafanaAlertmanagerMetrics) (*GrafanaAlertmanager, error) {
 	// TODO: Remove the context.
 	am := &GrafanaAlertmanager{
-		stopc:             make(chan struct{}),
-		logger:            log.With(logger, "component", "alertmanager", tenantKey, tenantID),
-		marker:            types.NewMarker(m.Registerer),
-		stageMetrics:      notify.NewMetrics(m.Registerer, featurecontrol.NoopFlags{}),
-		dispatcherMetrics: dispatch.NewDispatcherMetrics(false, m.Registerer),
-		peer:              peer,
-		peerTimeout:       config.PeerTimeout,
-		Metrics:           m,
-		tenantID:          tenantID,
-		externalURL:       config.ExternalURL,
+		stopc:                                   make(chan struct{}),
+		logger:                                  log.With(logger, "component", "alertmanager", tenantKey, tenantID),
+		marker:                                  types.NewMarker(m.Registerer),
+		stageMetrics:                            notify.NewMetrics(m.Registerer, featurecontrol.NoopFlags{}),
+		dispatcherMetrics:                       dispatch.NewDispatcherMetrics(false, m.Registerer),
+		peer:                                    peer,
+		peerTimeout:                             config.PeerTimeout,
+		Metrics:                                 m,
+		tenantID:                                tenantID,
+		externalURL:                             config.ExternalURL,
+		pipelineAndSateTimestampsMismatchAction: config.PipelineAndStateTimestampsMismatchAction,
```

Comment:
typo got copied here 😅 

```suggestion
		pipelineAndStateTimestampsMismatchAction: config.PipelineAndStateTimestampsMismatchAction,
```

Reply:
I can bet I fixed it!!